subprojects/logging/src/main/java/org/gradle/internal/logging/serializer/ProgressCompleteEventSerializer.java

```diff
@@ -28,13 +28,15 @@ public void write(Encoder encoder, ProgressCompleteEvent event) throws Exception
         encoder.writeSmallLong(event.getProgressOperationId().getId());
         encoder.writeLong(event.getTimestamp());
         encoder.writeString(event.getStatus());
+        encoder.writeBoolean(event.isFailed());
```

Comment:
Please ensure that this doesn't introduce any perf regressions as serialization can be expensive (we're probably okay, just checking).